timify-api/models/episode.js

```diff
@@ -1,62 +1,59 @@
 import { DataTypes } from 'sequelize';
 import { sequelize } from '../database.js';
-import { PodcastSeries } from './podcastseries.js';
 
 export const Episode = sequelize.define('Episode', {
     uuid: {
-        type: DataTypes.UUID,
-        defaultValue: DataTypes.UUIDV4,
-        primaryKey: true
-      },
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true
+    },
+    podcastUuid: {
+      type: DataTypes.UUID,
+      allowNull: false,
+      references: {
+        model: 'Podcast',
+        key: 'uuid'
+      }
+    },
+    hash: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
     name: {
-        type: DataTypes.STRING,
-        allowNull: false
+      type: DataTypes.STRING,
+      allowNull: false
     },
     description: {
-        type: DataTypes.TEXT,
-        allowNull: false
+      type: DataTypes.TEXT,
+      allowNull: false
     },
     imageUrl: {
-        type: DataTypes.STRING,
-        allowNull: false
+      type: DataTypes.STRING,
+      allowNull: false
     },
     datePublished: {
-        type: DataTypes.INTEGER,
-        allowNull: false
+      type: DataTypes.DATE,
+      allowNull: false
     },
-    hash: {
```

Comment:
What was this hash used for and why we need to delete it?

Reply:
i don't think i need it, so i left it out.

Reply:
Oh I see it above, never mind